Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -371,87 +366,27 @@ extension FunctionCallExprSyntax {
   }
 }
 
-/// Placeholder text must start with '<#' and end with
-/// '#>'. Placeholders can be one of the following formats:
-/// ```
-///   'T##' display-string '##' type-string ('##' type-for-expansion-string)?
-///   'T##' display-and-type-string
-///   display-string
-/// ```
-///
-/// NOTE: It is required that '##' is not a valid substring of display-string
-/// or type-string. If this ends up not the case for some reason, we can consider
-/// adding escaping for '##'.
-@_spi(SourceKitLSP)
-public enum EditorPlaceholderData {
-  case basic(text: Substring)
-  case typed(text: Substring, type: TypeSyntax)
+private struct EditorPlaceholderExpansionData {
+  let displayText: String
+  let type: TypeSyntax?
```

Comment:
Should this be `typeForExpansion` to match the member name in `RawEditorPlaceholderExpansionData`?